kernel/src/sched/nice.rs

```diff
@@ -19,6 +19,9 @@ define_atomic_version_of_integer_like_type!(Nice, try_from = true, {
 });
 
 impl Nice {
+    pub const MIN: Self = Nice::new(NiceRange::new(NiceRange::MIN));
+    pub const MAX: Self = Nice::new(NiceRange::new(NiceRange::MAX));
```

Comment:
Could you do me a favor by fixing the confusing name of `NiceRange`?

The name seems to suggest that it is a `Range<X>`, but it is not. I suggest the following changes.

```rust
pub type NiceValue = RangedI8<-20, 19>;

impl Nice {
    pub const fn value(&self) -> &NiceValue { ... }
}
```